kube/services/revproxy/gen3.nginx.conf/guppy-service.conf

```diff
@@ -1,4 +1,8 @@
           location /guppy/ {
+              if ($csrf_check !~ ^ok-\S.+$) {
+                return 403 "failed csrf check";
```

Comment:
maybe this could say "failed csrf check. Make sure to use guppy version >=x and data-portal version >=y"

Reply:
Good call, I can update this